pycord/interaction.py

```diff
@@ -23,10 +23,9 @@
 from functools import cached_property
 from typing import TYPE_CHECKING
 
-from .flags import MessageFlags
-
 from .embed import Embed
 from .errors import InteractionException
+from .flags import MessageFlags
```

Comment:
Reverts out of scope change.
```suggestion
from .flags import MessageFlags

from .embed import Embed
from .errors import InteractionException
```